### PR TITLE
CollectionViewCell

### PR DESCRIPTION
**Pull request by avhurst**
Create custom Collection view cell for frame. Set layout and wrote logic to set cell labels.

### CODE REVIEW

File: TurkeyTracker/TurkeyTracker/FrameCollectionViewCell.h [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+//  Copyright © 2016 Fastfoot Software. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "Frame.h"
+#import "Roll.h"
+
+@interface FrameCollectionViewCell : UICollectionViewCell
+
+@property (weak, nonatomic) IBOutlet UILabel *firstRoll;
```
Comment (brantonb) [inline TurkeyTracker/TurkeyTracker/FrameCollectionViewCell.h]: If you move these to top of FrameCollectionViewCell.m, then they won't be public. Basically, the owner of the collection view won't be able to reach in and update your controls. Generally speaking, keeping as much private as possible is considered good engineering.

You'll need to add this block to the top of the .m file:

    @interface FrameCollectionViewCell ()
    // UILabel properties go here
    @end

File: TurkeyTracker/TurkeyTracker/FrameCollectionViewCell.m [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        
+        self.firstRoll.text = [NSString stringWithCharacters:&circledChar length:1];
+        
+    } else {
+        self.firstRoll.text = [NSString stringWithFormat:@"%i", [[self.frame.rolls firstObject] getPinsStartingAsArray].count - [[self.frame.rolls firstObject] getPinsRemainingAsArray].count];
+    }
+    
+    
+    if (self.frame.rolls.count > 1) {
+        self.secondRoll.text = [NSString stringWithFormat:@"%i", [self.frame.rolls[1] getPinsStartingAsArray].count - [self.frame.rolls[1] getPinsRemainingAsArray].count];
```
Comment (brantonb) [inline TurkeyTracker/TurkeyTracker/FrameCollectionViewCell.m]: In the 10th frame, the second roll could also be a split.

**Comment by brantonb** [comment]
The ordering of the labels should be left to right. On frames 1-9, you should be using labels 2 & 3.

**Comment by avhurst** [comment]
Branch updated with current issues resolved.

**Pull request merged**